airbyte-server/src/main/resources/application.yml

```diff
@@ -105,6 +108,7 @@ datasources:
     maximum-pool-size: 10
     minimum-idle: 0
     idle-timeout: 600000
+    initialization-fail-timeout: -1   # Disable fail fast checking to avoid issues due to other pods not being started in time
```

Comment:
Curious, was this also failing in the server? Or are we adding this just in case. Trying to figure out how this change related to the rest of the changeset.

Reply:
@davinchia Yes, but I think that our pod retry policy made it not obvious, as the pod would keep trying and eventually the db pod would be available.

Reply:
got it, thanks